its_live_monitoring/src/sentinel2.py

```diff
@@ -158,7 +158,7 @@ def qualifies_for_sentinel2_processing(
         )
         return False
 
-    grid_square = item.properties['grid:code'][5:]
+    grid_square = item.properties['grid:code']
```

Comment:
Alternatively, we could just do
```suggestion
    grid_square = item.properties['grid:code'][5:].removeprefix("0")
```
here and not update the tile list. 

I am not sure which way to go, tbh.

Reply:
This PR makes the logic a little more straightforward in `its-live-monitoring` but makes using the `SENTINEL2_TILES_TO_PROCESS` harder in other contexts, and visa versa.

Reply:
I think adjusting the tile list makes the most sense -- that's actually what the query looks like, and going back to the ESA granule ID representation isn't terrible:
```python
import json
from pathlib import Path

SENTINEL2_TILES_TO_PROCESS = json.loads((Path('its_live_monitoring/src/sentinel2_tiles_to_process.json').read_text())

s2_tiles = [f'T{tile[5:]:0>5}' for tile in SENTINEL2_TILES_TO_PROCESS]
```